steps/50-lifecycle-lab1-marketplace-solution/.github/workflows/release-workflow.yml

```
@@ -9,6 +9,7 @@ jobs:
 
   release:
     runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/master'
```

```suggestion
    if: github.ref == 'refs/heads/main'
```
Master is deprecated

---

But not in semantic release 😄 (I have add a warning on this during the lab).